data/human/jobs.txt

```diff
@@ -4167,3 +4167,291 @@ mission "Republic Navy Advisory System [wildfire firefighters]"
 	on complete
 		payment 9000 85
 		dialog "You land at a makeshift landing site a few minutes flight from the blaze. The smoky haze that drifts into your ship when the firefighters open the hatch to depart makes your eyes and nose itch. As you watch your passengers heading toward a staging area nearby, you think about the inferno they will soon be voluntarily entering. A chirp from your communications system indicates your stipend of <payment> has been deposited, and you begin your atmospheric flight back to <planet>'s main spaceport."
+
+mission "Mineral Shortages (Silicon)"
+	name "Mineral shortages on <planet>"
+	job
+	repeat
+	deadline 7
```

Comment:
These deadlines feel like they might be too short, considering how the offer conditions don't restrict these jobs to require any specialized equipment. Have you tested these jobs, and are the smaller ones reasonable with a stock Sunder or laser interceptor?

Reply:
I have done some testing and the deadlines are pretty reasonable for most places in human space. I haven't exhaustively looked of course, so there probably are some places where you won't be able to complete the job. A stock Sunder should easily be able to complete the smaller jobs but a stock Sparrow would need a cargo expansion to fit the 30 tons.

Reply:
Okay, I think that's good then.